network-config-analyzer/nca.py

```diff
@@ -55,6 +57,13 @@ def _namespace_list_valid_path(path_location):
     return _valid_path(path_location, allow_ghe=True, allowed_platforms=['k8s'])
 
 
+def _resource_list_valid_path(path_location):
+    """
+    validation for paths of resources list in the command line
+    """
+    return _valid_path(path_location, allow_ghe=True, allowed_platforms=['k8s'])
```

Comment:
`allowed_platforms `should contain all options `['k8s', 'calico', 'istio']` , because resource list may refer to all resource types. However we do need to think about allowed combinations of multiple values for resource list, where involving both paths and platform names.. Also, determine some dependencies, e.g. if resource list is istio, then policies should be for istio , pods and namespaces from k8s. If resource is calico, then policies should be for calico, pods from calico and namespaces from k8s.

Reply:
thanks, I will start handling this